test/e2e/app-dir/segment-cache/prefetch-auto/next.config.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  experimental: {
+    ppr: true,
+    dynamicIO: true,
```

Comment:
This test I wrote doesn't rely on clientSegmentCache, I just put it in the same directory as those tests for now because it uses the same testing helpers/strategies. When clientSegmentCache lands in stable, I'll probably rename the directory to something like "prefetching".